### ISSUE TITLE
Output is too verbose with errors

### ISSUE THREAD

**Issue by ghost** [YoraiLevi/winutil_tweaks_cli#1]
Invoke-WebRequest https://raw.githubusercontent.com/YoraiLevi/winutil_tweaks_cli/master/winutil_cli.ps1 -OutFile winutil_cli.ps1
winutil_cli.ps1 -TweakNames WPFEssTweaksTele,WPFMiscTweaksDisableipsix,WPFEssTweaksHiber,WPFEssTweaksDVR,WPFEssTweaksServices,WPFEssTweaksLoc,WPFEssTweaksRemoveOnedrive,WPFEssTweaksWifi,WPFMiscTweaksDisplay,WPFMiscTweaksUTC,WPFEssTweaksRemoveEdge,WPFEssTweaksHome,WPFEssTweaksStorage,WPFEssTweaksAH,WPFEssTweaksOO,WPFEssTweaksTeredo

Tested didn't work

**Comment by ghost** [comment]
PS C:\Users\VIP> .\winutil_cli.ps1 -TweakNames WPFEssTweaksTele,WPFMiscTweaksDisableipsix,WPFEssTweaksHiber,WPFEssTweaksDVR,WPFEssTweaksServices,WPFEssTweaksLoc,WPFEssTweaksRemoveOnedrive,WPFEssTweaksWifi,WPFMiscTweaksDisplay,WPFMiscTweaksUTC,WPFEssTweaksRemoveEdge,WPFEssTweaksHome,WPFEssTweaksStorage,WPFEssTweaksAH,WPFEssTweaksOO,WPFEssTweaksTeredo
Transcript started, output file is C:\Users\VIP\AppData\Local\Temp\Winutil.log

Confirm
Are you sure you want to perform this action?
Performing the operation "Apply Tweak" on target "WPFEssTweaksTele".
[Y] Yes  [A] Yes to All  [N] No  [L] No to All  [S] Suspend  [?] Help (default is "Y"): a
Set-ScheduledTaskState : Scheduled Task Microsoft\Windows\Application Experience\Microsoft Compatibility Appraiser was not Found
At C:\Users\VIP\winutil_cli.ps1:681 char:21
+ ...             Set-ScheduledTaskState -TaskName $psitem.Name -State $psi ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Set-ScheduledTaskState

Set-ScheduledTaskState : Scheduled Task Microsoft\Windows\Application Experience\ProgramDataUpdater was not Found
At C:\Users\VIP\winutil_cli.ps1:681 char:21
+ ...             Set-ScheduledTaskState -TaskName $psitem.Name -State $psi ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Set-ScheduledTaskState

Set-ScheduledTaskState : Scheduled Task Microsoft\Windows\Autochk\Proxy was not Found
At C:\Users\VIP\winutil_cli.ps1:681 char:21
+ ...             Set-ScheduledTaskState -TaskName $psitem.Name -State $psi ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException



and so on red errors

**Comment by ghost** [comment]
[Winutil.log](https://github.com/YoraiLevi/winutil_tweaks_cli/files/13650114/Winutil.log)

**Comment by YoraiLevi** [comment]
I'll take a look at it soon enough.
Meanwhile, if you have any output or system information please share that here.

**Comment by ghost** [comment]
The GUI variant works OK and shows good output messages, however the CLI variant returned too much red stuff. Hulk mode. 🤯

**Comment by YoraiLevi** [comment]
Yes the cli doesn't filter out the error messages. a lot of tweaks come with what i would consider bugs, and error out silently. I haven't looked into it too much because it was not the goal at the time.
If the tweak works regardless of the red output it's working.
